pkg/graph/resolvers/resolver_test.go

```diff
@@ -408,6 +408,32 @@ func (suite *ResolverSuite) createMilestoneCommon(
 	return milestone
 }
 
+func (suite *ResolverSuite) getMilestonesWithNoLinkedSolution(
```

Comment:
I don't think this function is needed. These functions here are generally are setting up test data. This is abstracting a fetch. I think reworking the unit test to use `VerifyLoaders` would negate the use of this as well.